src/ShellCheck/Data.hs

```diff
@@ -156,6 +156,8 @@ shellForExecutable name =
         "sh"    -> return Sh
         "bash"  -> return Bash
         "bats"  -> return Bash
+        "busybox sh"  -> return Busyboxsh
+        "busybox ash"  -> return Busyboxsh
         "dash"  -> return Dash
         "ash"   -> return Dash -- There's also a warning for this.
```

Comment:
Any thoughts on whether Dash or BusyboxSh is a better match for `ash`?

Reply:
Dash is probably the more canonical Ash. I wouldn't change that.